components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -293,4 +293,37 @@ - (void)testManagerForwardsButtonProperties {
   XCTAssertEqualObjects(UIColor.redColor, actionButton.inkColor);
 }
 
+- (void)testTraitCollectionDidChangeCalledWhenTraitCollectionChanges {
+  // Given
+  MDCSnackbarMessage *message = [MDCSnackbarMessage messageWithText:@"test"];
+
+  // When
+  [self.manager showMessage:message];
```

Comment:
You shouldn't need to show a message to test this API.  Please just call the method on the view directly.